### ISSUE TITLE
Datadog Profiling Python 3.11 Support

### ISSUE THREAD

**Issue by corydolphin** [DataDog/dd-trace-py#5523]
### Summary of problem

Python 3.11 was released December 2022, e.g roughly four months ago. Is there a timeline for support for Datadog's Python profiler?

I initially went to the closed issue referenced here https://github.com/DataDog/dd-trace-py/issues/4149

```
RuntimeError: Profiling is not yet compatible with Python 3.11. See tracking issue for more details: https://github.com/DataDog/dd-trace-py/issues/4149
```

### Which version of dd-trace-py are you using?
1.11.1

### Which libraries and their versions are you using?

<details>
  <summary>`pip freeze`</summary>
  
</details>

**Comment by sanchda** [comment]
:wave: Thanks for reporting.  Support for Python 3.11 is available in `ddtrace` 1.11.1.  Are you running into problems with the profiler?

**Comment by mabdinur** [comment]
Closing this issue due to inactivity. The latest release of ddtrace supports python 3.11. 

Feel free to open a new issue to report bugs and/or incompatibilities.
